gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -65,9 +65,9 @@ public class KafkaAvroJobStatusMonitor extends KafkaJobStatusMonitor {
   private Meter messageParseFailures;
 
   public KafkaAvroJobStatusMonitor(String topic, Config config, int numThreads,
-      JobIssueEventHandler jobIssueEventHandler)
+      JobIssueEventHandler jobIssueEventHandler, boolean instrumentationEnabled)
```

Comment:
instrumentation is vague, but even there suggests something different, such as us measuring how fast the class operates as it handles messages.  why not call it what it is--`shouldEmitObervabilityEvents`?

Reply:
I'm going to leave instrumentationEnabled only for incrementing the metrics of the meters/gauges, and the whether or not the class is configured to emit events will be handled separately.